test/e2e/fleetautoscaler_test.go

```diff
@@ -523,10 +519,12 @@ func TestTlsWebhook(t *testing.T) {
 		// if we could not create the autoscaler, their is no point going further
 		assert.FailNow(t, "Failed creating autoscaler, aborting TestTlsWebhook")
 	}
-	fa := getAllocation(flt)
-	fa, err = alpha1.FleetAllocations(defaultNs).Create(fa.DeepCopy())
-	assert.Nil(t, err)
-	assert.Equal(t, stablev1alpha1.GameServerStateAllocated, fa.Status.GameServer.Status.State)
+	gsa := getAllocation(flt)
+	gsa, err = framework.AgonesClient.AllocationV1alpha1().GameServerAllocations(flt.ObjectMeta.Namespace).Create(gsa)
+	if !assert.Nil(t, err) {
+		assert.FailNow(t, "gameserverallocation could not be created")
```

Comment:
log err?

Reply:
Switch to `assert.NoError` which will output the error message if it fails. `NoError` is a better choice (which i didn't realise until recently).